### PR TITLE
Various consistency changes

### PR DESCRIPTION
**Pull request by benjay10**
Mainly some consistency things:

* Using cache instead of resources in all calls from the dispatcher.
  - *Why was this not the case yet? Am I overlooking something?*
* Newlines in all docker-compose files (1 file didn't have the same whitespacing as the others).
* Removed version numbers from the docker-compose files, because they have been deprecated.

### CODE REVIEW

File: config/dispatcher/dispatcher.ex [hunk 1]
```diff
@@ -134,15 +134,15 @@ defmodule Dispatcher do
   end
 
   get "/public-services/*path" do
-    forward conn, path, "http://resource/public-services/"
+    forward conn, path, "http://cache/public-services/"
```
Comment (mirdono) [inline config/dispatcher/dispatcher.ex:137]: **Issue:** In the docs an [ADR](https://github.com/lblod/app-lpdc-digitaal-loket/blob/master/docs/adr/0005-do-not-cache-instantie-overview.md) is documented on why the cache service is not used here. I assume this ADR is still relevant and think that we should **not** change this (and related paths) to the cache service, at least not without further analysis.

**Review by mirdono — CHANGES_REQUESTED** [review]
I left a blocking comment concerning the switch to the cache service. I am fine with the other changes concerning removed newlines and version.

File: config/dispatcher/dispatcher.ex (hunk 1 above)
Comment (benjay10) [reply config/dispatcher/dispatcher.ex:137]: Aha okay, I did not know about the ADRs. I will revert the routes for `public-services` and `concept-display-configurations` back to `mu-cl-resources` directly.

**Review by mirdono — APPROVED** [review]
Looks good to me.

Apologies for the delay, it seems I lost track of this PR since the last changes.

**Pull request merged**